docs/platforms/dotnet/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -28,6 +28,18 @@ Upon sending a request to `https://example.com`, the instrumented HTTP client wi
 - Populate the `sentry-trace` header on the request. This allows peer service to start a transaction by linking it to the current (assuming it's also instrumented with Sentry).
 - Start a span named `GET https://example.com` which will track the corresponding HTTP operation on the current transaction.
 
+To avoid depleting performance quota, Sentry will only create request spans when there is an active transaction on the scope. If there is no active transaction then you will need to create one before making an HTTP request, for request tracing to work:
+
```

Comment:
```suggestion
To avoid depleting performance quota, Sentry will only create request spans if there's an active transaction on the scope. If there's no active transaction, you'll need to create one before making an HTTP request for request tracing to work:

```